gui/control_main.py

```diff
@@ -353,6 +353,8 @@ def createSampleTab(self):
         self.closeShutterButton = QtWidgets.QPushButton("Close Photon Shutter")
         self.closeShutterButton.clicked.connect(self.closePhotonShutterCB)
         self.closeShutterButton.setStyleSheet("background-color: red")
+        self.parkRobotButton = QtWidgets.QPushButton("Park Robot")
```

Comment:
you need to find a way to make these activations and deactivations clean for NYX (or whatever other state you want to call it). these must not appear in code that is to run on AMX/FMX.
this problem happens throughout the GUI code - both the deactivations and new buttons